docs/en/ingest-management/elastic-agent/configuration/providers/elastic-agent-providers.asciidoc

```diff
@@ -0,0 +1,103 @@
+[discrete]
```

Comment:
```suggestion
```

Reply:
Ah, I see the problem. The discrete tag basically forces the build to ignore chunking for that section. When you commit my suggestion, the docs build will run again, and the preview will be updated.

The preview link doesn't change. In fact, if you're ever in a situation where you don't see a link for the docs preview, just look at the console output and search for `preview`. You'll find the link in the log.

Reply:
Thank you!